### DISCUSSION TITLE
"Creating a Pyodide package" - new package PR merged, but not included in latest release?

### DISCUSSION THREAD

**Discussion by michaelweinold** [pyodide/pyodide#4081] [Q&A] [answered]
I added a new package to Pyodide in June, which was merged by @hoodmane:

 - https://github.com/pyodide/pyodide/pull/3897

However, it seems that it has not been included in the [Pyodide package list](https://pyodide.org/en/stable/usage/packages-in-pyodide.html) of the latest release in July (0.23.4).

Unfortunately, I did not find any documentation on how new packages are added to releases.

Is there further action required on my part?
Will the package be added to the next version (if so, when will it be released)?

**Comment by ryanking13** [comment, marked as answer]
It will be added in the next major release 0.24.0 (we are planning to release 0.24.0 soon). We normally only include bugfixes in the minor release so peewee was not included in the 0.23.4.
